Doc/library/gzip.rst

```diff
@@ -222,6 +222,8 @@ Once executed the :mod:`gzip` module keeps the input file(s).
 .. versionchanged:: 3.8
 
    Add a new command line interface with a usage.
+   By default, when you will execute the CLI, the default compression level is 6,
```

Comment:
I don't understand if the default is 6 or 9.

Reply:
normally, it's 6, it's the tradeoff. I suppose there is a ambiguity with `--best` selected by default, I am going to update the documentation.

Reply:
the default on the command line is 6, not when you will use the API.